portugues/colaborando.md

```diff
@@ -25,13 +25,12 @@ A sua linha telefônica está uma droga hoje e você tem que baixar a ultima ver
 
 [Você liga para a companhia telefônica](telefonica/ocupado.md)
 
-O Git te parece ser util, mas como você ainda não vive no século XXI, você está mais preocupado com esse tal de Windows.
+O Git lhe parece ser util, mas como você ainda não vive no século XXI, você está mais preocupado com esse tal de Windows.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.